godtools/App/Share/Domain/UseCases/GetDeviceLanguageUseCase/GetDeviceLanguageUseCase.swift

```diff
@@ -9,20 +9,36 @@
 import Foundation
 
 class GetDeviceLanguageUseCase {
+        
+    private let getLanguageUseCase: GetLanguageUseCase
```

Comment:
`GetDeviceLanguageUseCase` doesn't seem to be using `GetLanguageUseCase`.  Should it?

Reply:
I was contemplating that as well.  There are some differences between them though.
- The LanguageDomainModel is mapped from mobile content api languages with applied business logic.
- Where as DeviceLanguageDomainModel is mapped from the OS System Device languages, in which we aren't applying any special business logic at this time.  We mostly use this for enabling specific device language features in the app.  

There is a relationship between the 2 of these where the language codes from our languages in mobile content api match a Locale identifier.

For now, let's just keep them distinct.  This way we can grab the current device language without depending on the mobile content languages api.

Reply:
Makes sense to me.  Can we remove this useCase dependency then?